.github/actions/bazel/bin/bazel

```diff
@@ -58,6 +58,10 @@ fi
 if [[ $bazel_command == "build" ]] || [[ $bazel_command == "test" ]]; then
     command_timestamp=$(date +%s)
     bazel_args+=(
+
+        # XXX: the latest command will overwrite the file
```

Comment:
we should instead use the `command_timestamp` and (when converting to CSV) look up all `execution_log-*.json` files